compiler/vm.nim

```diff
@@ -523,8 +523,14 @@ proc rawExecute(c: PCtx, start: int, tos: PStackFrame): TFullReg =
   # Used to keep track of where the execution is resumed.
   var savedPC = -1
   var savedFrame: PStackFrame
-  var regs: seq[TFullReg] # alias to tos.slots for performance
-  move(regs, tos.slots)
+  when defined(gcArc):
```

Comment:
```suggestion
  when defined(gcArc) or defined(gcOrc):
```

Reply:
`--gc:orc` should actually define `gcArc` but doesn't right now, see #17346

Reply:
There is ``gcDestructors`` that should be used.